README.md

```diff
@@ -85,6 +85,38 @@ Run `go get -t ./...` in order to install project's dependencies.
 
 Run tests with `FAUNA_ROOT_KEY="your-cloud-secret" go test ./...`.
 
+Tests can also be run via a Docker container. The `Dockerfile.test` file
+creates an image that automatically runs the tests against a target FaunaDB
+cluster. You can run tests this way via the following steps:
+
+1. Build the image via
+   `docker build -f Dockerfile.test --build-arg RUNTIME_IMAGE=<image> .`, where
```

Comment:
Lets add a `-t` argument to the example so that we can tag the build and use that name in the subsequent `docker run` examples rather than `<built-image>` placeholders.

Reply:
What tag should I use?

Reply:
just a logical name for the image. `fauna-go-driver-test:latest`?

Reply:
or `faunadb-go-test:latest`, idk